signal_scanner_bot/messages.py

```diff
@@ -168,4 +168,4 @@ def process_twitter_message(status: Status) -> None:
 
     # On the off chance a message is an empty string just skip sending
     if message:
-        env.TWITTER_TO_SIGNAL_QUEUE.put(message, block=False)
+        env.TWITTER_TO_SIGNAL_QUEUE.put_nowait(message)
```

Comment:
Nice! I think in asynchronous methods, we'll want to `await` various `Queue` methods when possible

Reply:
I think is going to be a problem using the current tweepy library. I think because the stream.filter() function doesn't return anything and all actions taken are required to be run through the Listener class, we can't actually get any data back into the async twitter_to_queue function, and thus can't use the await keyword.

I'm confused as hell though so I may just be completely wrong and missing something.

Reply:
Ah no you're totally right, sorry that's me being scatterbrained. You're spot on and this makes sense for now!

Reply:
Ok cool I really wasn't sure haha. Do you think we should merge this branch in before moving on to trying to rebase on to the new library?

I still want to do a longer term test to see if it:
A - fixes the issue we're trying to fix
B - doesn't run into problems with high tweet volume
C - doesn't have any other unforeseen issues

before doing the merge. I'm still not exactly sure how we're gonna do the test, probably add a new group to run in parallel to the other one so it doesn't inundate the admin channel.